tests/detect-flowbits/test.yaml

```diff
@@ -0,0 +1,16 @@
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 13
+    match:
+      event_type: alert
```

Comment:
I _think_ we need more details here, as we have many rules and in the end it's not very clear which ones are matching, nor how. Some input (based on the unittests) to help deciding on that, hopefully...

Test06, Test07 and Test08:
not sure how to go about these... Maybe if we used also `noalert` to the first `flowbists:set`, then added a `content` keyword for the rule with `flowbits:toggle` and made sure the test expected that one to match for an alert? 
Test09:
expects matches from 1 and 2, but no match from 4
Test11:
expects matches from 1, 2, 3 but no match for 5
Test10:
expects matches from 1, 2, 3 and 4